terra-workspace-manager/sa.tf

```diff
@@ -18,10 +18,12 @@ resource "google_project_iam_member" "sqlproxy" {
 
 locals {
   app_sa_roles = [
-    # Allow exporting metrics, profiling, and tracing for monitoring.
-    "roles/monitoring.editor",
     "roles/cloudprofiler.agent",
     "roles/cloudtrace.agent",
+    # Allow exporting metrics, profiling, and tracing for monitoring.
```

Comment:
now this comment follows the roles for profiling and tracing. Can we keep the comment next to the roles?